modules/support_ticket/support_ticket.tokens.inc

```diff
@@ -133,14 +131,13 @@ function support_ticket_tokens($type, $tokens, array $data, array $options, Bubb
           $translation = \Drupal::entityManager()->getTranslationFromContext($support_ticket, $langcode, array('operation' => 'support_ticket_tokens'));
           if ($translation->hasField('body') && ($items = $translation->get('body')) && !$items->isEmpty()) {
             $item = $items[0];
-            $field_definition = \Drupal::entityManager()->getFieldDefinitions('support_ticket', $support_ticket->bundle())['body'];
             // If the summary was requested and is not empty, use it.
             if ($name == 'summary' && !empty($item->summary)) {
```

Comment:
Dunno why the empty check is like this, but if it's wrong the problem is upstream :-/